geotrellis-tiling/src/main/scala/geotrellis/geotrellis-tiling/LayoutDefinition.scala

```diff
@@ -14,12 +14,11 @@
  * limitations under the License.
  */
 
-package geotrellis.spark.tiling
+package geotrellis.tiling
```

Comment:
the package name is `geotrellis.tiling`, however the full path is `geotrellis/geotrellis-tiling/src/main/scala/geotrellis/geotrellis-tiling/`. 

Should be `geotrellis/tiling/src/main/scala/geotrellis/tiling/`. The package name should be the same as the directory name.